src/snap/plugin.cc

```diff
@@ -52,7 +53,46 @@ Plugin::Meta::Meta(Type type, std::string name, int version) :
                      concurrency_count(5),
                      exclusive(false),
                      cache_ttl(std::chrono::milliseconds(500)),
-                     strategy(Strategy::LRU) {}
+                     strategy(Strategy::LRU),
+                     tls_enabled(false),
+                     tls_certificate_key_path(""),
+                     tls_certificate_crt_path(""),
+                     tls_certificate_authority_path(""),
+                     tls_cipher_suites("")
+                     { const char* tls_env = getenv("TLS_ENABLED");
+                       if (tls_env != nullptr && strcmp(tls_env, "false") != 0) {
+                         std::stringstream message_template{};
+                         message_template << "Encryption enabled but ";
+
+                         const char* taoCS = getenv("TAOCS");
+                         const char* taoCC = getenv("TAOCC");
+                         const char* taoCK = getenv("TAOCK");
+                         const char* taoCA = getenv("TAOCA");
+
+                         if (taoCS != nullptr) {
+                           this->tls_cipher_suites.assign(taoCS);
+                         }
+
+                         if (taoCC == nullptr) {
+                           message_template << "TAOCC not set.";
+                           throw PluginException(message_template.str());
+                         }
+                         if (taoCK == nullptr) {
+                           message_template << "TAOCK not set.";
+                           throw PluginException(message_template.str());
+                         }
+                         if (taoCA == nullptr) {
+                           message_template << "TAOCA not set.";
+                           throw PluginException(message_template.str());
+                         }
+
+                         this->tls_enabled = true;
+
+                         this->tls_certificate_crt_path.assign(taoCC);
+                         this->tls_certificate_key_path.assign(taoCK);
+                         this->tls_certificate_authority_path.assign(taoCA);
+                       }
+                     }
```

Comment:
Your bracket indentation here should resemble the other methods. In this case, against the left side.